### PR TITLE
Adding note about OS processor version and OpenSSL

### PR DESCRIPTION
**Pull request by lidiazuin**

### CODE REVIEW

**Review by davidfauth — APPROVED** [review]
Changes look good to me.

File: modules/ROOT/pages/security/ssl-framework.adoc [hunk 1]
```diff
@@ -26,7 +26,8 @@ Follow these steps to use OpenSSL:
 Dependencies can be downloaded from https://netty.io/wiki/forked-tomcat-native.html.
 Which dependencies you need depends upon the Neo4j version.
 Each version of Neo4j ships with a version of Netty and Netty requires specific tcnative versions.
-See table below.
+Make sure to install the version that matches your OS processor.
+For more details, see the table below.
```
Comment (renetapopova) [inline modules/ROOT/pages/security/ssl-framework.adoc:30]: It looks good! I would change `below` to an actual link to the table and add an anchor and a caption to the table.

**Review by renetapopova — COMMENTED** [review]
Thanks, @lidiazuin. I added one editorial suggestion.

File: modules/ROOT/pages/security/ssl-framework.adoc [hunk 2]
```diff
@@ -26,7 +26,8 @@ Follow these steps to use OpenSSL:
 Dependencies can be downloaded from https://netty.io/wiki/forked-tomcat-native.html.
 Which dependencies you need depends upon the Neo4j version.
 Each version of Neo4j ships with a version of Netty and Netty requires specific tcnative versions.
-See table below.
+Make sure to install the version that matches your OS processor.
+For more details, see the <<table, table>>.
```
Comment (renetapopova) [inline modules/ROOT/pages/security/ssl-framework.adoc:30]: You are so sweet. I meant something like this, but feel free to change it if you'd like:
```suggestion
For more details, see the <<table, Netty support per Neo4j version>>.
```

File: modules/ROOT/pages/security/ssl-framework.adoc [hunk 3]
```diff
@@ -37,6 +38,7 @@ Most supported versions of Neo4j use Netty 4.1.77.Final, which requires tcnative
 Only Neo4j 3.5 still uses older versions of Netty.
 See the table below for detailed information:
 
+[[table]]
```
Comment (renetapopova) [inline modules/ROOT/pages/security/ssl-framework.adoc:41]:
```suggestion
[[table]]
.Netty support per Neo4j version
```

**Review by renetapopova — APPROVED** [review]

File: modules/ROOT/pages/security/ssl-framework.adoc (hunk 2 above)
Comment (lidiazuin) [reply modules/ROOT/pages/security/ssl-framework.adoc:30]: ah ok 😅 thanks for the clarification!!

**Pull request merged**